cdhweb/blog/migrations/0005_auto_20180319_1212.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2018-03-19 16:12
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import mezzanine.core.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blog', '0004_blogpost_is_featured'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blogpost',
+            name='featured_image',
+            field=mezzanine.core.fields.FileField(blank=True, help_text='Appears on the homepage carousel when post is featured.', max_length=255, null=True, verbose_name='Featured Image'),
```

Comment:
If you still can, would be nice if you would remove migrations 4 and 5 and regenerate a single one with the nice name that you used for the first one so we don't get an extra unnamed migration just for changes in help text.